force-app/main/default/classes/PMT_ValidationService.cls

```diff
@@ -148,6 +148,10 @@ public inherited sharing class PMT_ValidationService {
     }
 
     private void validateRefund(npe01__OppPayment__c payment) {
+        if (String.isNotBlank(payment.Elevate_Payment_ID__c) && config.isIntegrationEnabled()) {
```

Comment:
What happens here if Elevate_Payment_ID__c, but something is currently wrong with the Elevate config. Is the transaction permanently messed up?

Reply:
The reason the validation for Elevate is removed is because of the following:
- Elevate refund will always be populated by the System, it will never be user input.
- We almost ran into dependency issues in the 240 release because the validation service is checking extra fields that the Connector package does not know before the 240 release and we cannot guarantee the update timing for both packages.